README.md

```diff
@@ -33,12 +33,12 @@ pip install elastic-opentelemetry
 ## Usage
 
 Our distribution does not install any instrumentation package by default, instead it relies on the
-`opentelemetry-bootstrap` command to scan the installed packages and install the available instrumentation.
+`edot-bootstrap` command to scan the installed packages and install the available instrumentation.
```

Comment:
would this be useful?

```suggestion
`edot-bootstrap` command to scan the installed packages and install the available instrumentation, prioritizing EDOT variants when available.
```

Reply:
I think this is more about preferring than prioritizing